### DISCUSSION TITLE
Incorrect links in Lesson 14 - Vitalik's documents

### DISCUSSION THREAD

**Discussion by AbhiSharma1210** [Cyfrin/foundry-full-course-cu#1424] [General]
Hi, 
As of writing this post, the links in Lesson 14 DAO section which redirects to Vitalik's documents seems outdated/incorrect.
Following links are mentioned. 
[Plutocracy is bad](https://vitalik.ca/general/2018/03/28/plutocracy.html)
[DAOs are not corporations](https://vitalik.ca/general/2022/09/20/daos.html)

should be updated to the following instead.
1. Plutocracy is bad - https://vitalik.eth.limo/general/2018/03/28/plutocracy.html
2. DAOs are not corporations - https://vitalik.eth.limo/general/2022/09/20/daos.html

Thank you

**Comment by EngrPips** [comment]
Thank you so much for pointing that out, it would be looked into.
